src/main/java/com/openshift/internal/restclient/model/BuildConfig.java

```diff
@@ -142,6 +200,14 @@ public void setOutput(DockerImageURI imageUri){
 		return null;
 	}
 
+	private void storeEnvironmentVars(final String [] key, ModelNode root, Map<String, String> envs){
```

Comment:
@jcantrill Should be handled in this commit: https://github.com/aslakknutsen/openshift-restclient-java/commit/9b738fb5a8751774f6c450a6026a68ceba057974 

Named them setEnvMap and getEnvMap to differentiate between Kubernetes EnvVar(using name and value) type used here and just  JSON Maps used in e.g. Labels (asMap())